packages/server/tests/localAcceptance.env

```diff
@@ -30,3 +30,4 @@ WRITE_SNAPSHOT_ON_MEMORY_LEAK=false
 HBAR_RATE_LIMIT_TINYBAR=5000000000# 50 HBARs
 HBAR_RATE_LIMIT_DURATION=80000# 80 seconds
 HBAR_RATE_LIMIT_BASIC=4000000000# 40 HBARs
+HBAR_SPENDING_PLANS_CONFIG_FILE=./packages/server/tests/testSpendingPlansConfig.json
```

Comment:
**nit:** `findConfig` will do it's job and we don't need to specify the full path here:
```suggestion
HBAR_SPENDING_PLANS_CONFIG_FILE=testSpendingPlansConfig.json
```

Reply:
synced offline and findout findConfig returns undefined with fileName and works fine with filePath.